models/ed/DESCRIPTION

```diff
@@ -24,11 +24,17 @@ Imports:
     stringr(>= 1.1.0),
     udunits2 (>= 0.11),
     XML (>= 3.98-1.4),
-    hdf5r
+    hdf5r,
```

Comment:
so have we dropped the old HDF5 package now?

Reply:
We may need to update the install.depends script to account for this change.

Reply:
Yes, `rhdf5` is more difficult to install (not on CRAN), and may not be actively maintained anymore. This one is actively maintained, and IMHO provides a more convenient interface.